digitalpublicgoods/vaccineledger.json

```diff
@@ -0,0 +1,102 @@
+{
+  "name": "VaccineLedger",
+  "clearOwnership": {
+    "isOwnershipExplicit": "Yes",
+    "copyrightURL": "https://gitlab.com/statwig-public/theledger/-/blob/development/LICENSE"
+  },
+  "platformIndependence": {
+    "mandatoryDepsCreateMoreRestrictions": "No",
+    "isSoftwarePltIndependent": "",
+    "pltIndependenceDesc": ""
+  },
+  "documentation": {
+    "isDocumentationAvailable": "Yes",
+    "documentationURL": [
+      "https://gitlab.com/statwig-public/theledger",
+      "https://gitlab.com/statwig-public/theledger/-/blob/development/docs/VaccineLedger-_Use_Case___Functional_Features_.pdf",
+      "https://gitlab.com/statwig-public/theledger/-/blob/development/README.md"
+    ]
+  },
+  "NonPII": {
+    "collectsNonPII": "Yes",
+    "checkNonPIIAccessMechanism": "Yes",
+    "nonPIIAccessMechanism": "This data is stored on Blockchain and can be accessible through REST APIs.\nThe core idea of blockchain is collaboration based on shared data ownership, governance and  operations. The major difference from centralized system architectures is that peers,  collaborating in a network, can continue using their own systems and technology stack without  the need to integrate them in the classical way. Instead, blockchain provides a shared data  layer that allows these systems to interoperate on top and acts as the single truth. \n\nAs a result, the network has no single authority, which owns the data or controls the systems.  The decentralization of the ledger means a decentralization of power.\n\nLink to Evidence: https://gitlab.com/statwig-public/theledger/-/blob/development/docs/VaccineLedger_-_Data_Confidentiality.pdf"
+  },
+  "privacy": {
+    "isPrivacyCompliant": "Yes",
+    "privacyComplianceList": [
+      "https://gitlab.com/statwig-public/theledger/-/blob/development/docs/Server.pdf"
+    ],
+    "adherenceSteps": [
+      "Privacy policy - https://gitlab.com/statwig-public/theledger/-/blob/development/docs/Privacy_policy_of_VaccineLedger__1_.pdf",
+      "https://gitlab.com/statwig-public/theledger/-/blob/development/docs/Server.pdf"
+    ]
+  },
+  "standards": {
+    "supportStandards": "Yes",
+    "standardsList": [
+      "WaSP",
+      "HTML",
+      "CSS",
+      "ECMAScript",
+      "JSON",
+      "REST"
+    ],
+    "evidenceStandardSupport": [
+      "https://gitlab.com/statwig-public/theledger/-/blob/development/docs/Standards"
+    ],
+    "implementBestPractices": "Yes",
```

Comment:
I think it is debatable whether their documentation is considered as implementing best practices, both for the general inaccessibility of the documentation as PDFs embedded into a git repository and for the note @iperdomo added in his review.

Reply:
According to the [DPG standard](https://github.com/DPGAlliance/DPG-Standard/blob/main/standard.md), their existing documentation passes because it allows a technical person unfamiliar with the project to launch and run the software, hence any additions would help as recommendations not requirements to become a DPG. See [indicator #5](https://github.com/DPGAlliance/DPG-Standard/blob/main/standard.md). Thanks.